jboss-eap7.3-clustered-multivm-rhel8/README.md

```diff
@@ -48,7 +48,8 @@ Following is the Architecture:
 
 To learn more about the JBoss Enterprise Application Platform, visit [Red Hat EAP 7.3 documentation](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.3/).
 
-## Subscriptions and Costs - Read the instructions in their entirety before deploying 
+## Subscriptions and Costs
+Read the instructions in their entirety before deploying 
```

Comment:
We have completely changed the structure of “Subscriptions and Costs” section in the new Draft in https://github.com/SpektraSystems/redhat-mw-cloud-quickstart/blob/master/jboss-eap-standalone-rhel7/README.md. Please review and let us know the feedback, so that we can replicate the same format here.